crates/sui-core/src/validator_client_monitor/monitor.rs

```diff
@@ -155,7 +164,20 @@ where
                 }
             }
 
-            self.update_cached_scores();
+            // If the checks are disabled, then we skip updating the cached scores.
```

Comment:
If we update the scores, then it's highly possible that we crystalise some non-representative scores as we'll have mostly healthcheck data. As the FN for every request is choosing one validator out of the top scoring `K = (commitee_num/3)` , this means that we'll have a large amount of nodes that will have to wait for the ping requests in order to start re-scoring properly. That's why we want to give some time before we calculate the scores for the first time.